cpp/kiss_icp/3rdparty/sophus/sophus.cmake

```diff
@@ -25,6 +25,12 @@ set(SOPHUS_USE_BASIC_LOGGING ON CACHE BOOL "Don't use fmt for Sophus libraru")
 set(BUILD_SOPHUS_TESTS OFF CACHE BOOL "Don't build Sophus tests")
 set(BUILD_SOPHUS_EXAMPLES OFF CACHE BOOL "Don't build Sophus Examples")
 
-# TODO: after https://github.com/strasdat/Sophus/pull/502 gets merged go back to mainstream
-FetchContent_Declare(sophus SYSTEM URL https://github.com/nachovizzo/Sophus/archive/refs/tags/1.22.11.tar.gz)
+FetchContent_Declare(
+  sophus
+  SYSTEM
+  GIT_REPOSITORY https://github.com/strasdat/Sophus.git
+  GIT_TAG 1.22.10
+  PATCH_COMMAND git apply --ignore-space-change --ignore-whitespace ${CMAKE_CURRENT_LIST_DIR}/sophus.patch
```

Comment:
This patch also relies on git, I think there is a UNIX way of doing this without invoking git